src/ontology/mondo.Makefile

```diff
@@ -264,7 +264,12 @@ reports/%-rare-diseases.tsv: $(ONT)-base.owl reports/%-rare-diseases.txt
 rare-disease-reports: reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv
 	python ../scripts/filter_rare_disease_list.py reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv reports/added-rare-disases.tsv reports/removed-rare-diseases.tsv
 
-	
+#############################################
+##### One-time scripts ######################
```

Comment:
## One-time scripting
@matentzn There are enough steps here that I want to codify this rather than do things manually. I also want you/others to be able to check my work too.

I think since this will be a one time thing, perhaps what we can do is at the end of this PR, I can add another commit to remove any make commands / intermediate Python scripts, and leave only the updated `mondo-edit.obo` and the added robot template. What do you think?